config/routes.js

```diff
@@ -46,62 +46,73 @@ module.exports.routes = {
+  '/api/game/untargetedOneOff': 'game/untargeted-one-off',
+  '/api/game/targetedOneOff': 'game/targeted-one-off',
+  '/api/game/counter': 'game/counter',
+  '/api/game/resolve': 'game/resolve',
+  '/api/game/resolveFour': 'game/resolve-four',
+  '/api/game/resolveThree': 'game/resolve-three',
   // Resolving Sevens
-  '/game/seven/points': 'game/seven/points',
-  '/game/seven/faceCard': 'game/seven/face-card',
-  '/game/seven/scuttle': 'game/seven/scuttle',
-  '/game/seven/jack': 'game/seven/jack',
-  '/game/seven/untargetedOneOff': 'game/seven/untargeted-one-off',
-  '/game/seven/targetedOneOff': 'game/seven/targeted-one-off',
-  '/game/over': 'game/game-over',
-  '/game/concede': 'game/concede',
-  '/game/stalemate': 'game/stalemate',
-  '/game/reject-stalemate': 'game/stalemate-reject',
-  '/game/rematch': 'game/rematch',
-  '/game/join-rematch': 'game/join-rematch',
-  '/game/chat': 'game/chat',
-  '/game/gameData': 'game/game-data',
+  '/api/game/seven/points': 'game/seven/points',
+  '/api/game/seven/faceCard': 'game/seven/face-card',
+  '/api/game/seven/scuttle': 'game/seven/scuttle',
+  '/api/game/seven/jack': 'game/seven/jack',
+  '/api/game/seven/untargetedOneOff': 'game/seven/untargeted-one-off',
+  '/api/game/seven/targetedOneOff': 'game/seven/targeted-one-off',
+  '/api/game/over': 'game/game-over',
+  '/api/game/concede': 'game/concede',
+  '/api/game/stalemate': 'game/stalemate',
+  '/api/game/reject-stalemate': 'game/stalemate-reject',
+  '/api/game/rematch': 'game/rematch',
+  '/api/game/join-rematch': 'game/join-rematch',
+  '/api/game/chat': 'game/chat',
+  '/api/game/gameData': 'game/game-data',
   // DEVELOPMENT ONLY
-  '/game/stackDeck': 'game/stack-deck',
-  '/game/loadFixture': 'game/load-fixture',
-  '/game/clear': 'game/clear-game',
+  '/api/game/stackDeck': 'game/stack-deck',
+  '/api/game/loadFixture': 'game/load-fixture',
+  '/api/game/clear': 'game/clear-game',
 
   // Testing helpers
-  '/test/wipeDatabase': 'TestController.wipeDatabase',
-  '/test/badSession': 'TestController.setBadSession',
-  '/test/loadSeasonFixture': 'TestController.loadSeasonFixture',
-  '/test/loadMatchFixtures': 'TestController.loadMatchFixtures',
-  '/test/loadFinishedGameFixtures': 'TestController.loadFinishedGameFixtures',
+  '/api/test/wipeDatabase': 'TestController.wipeDatabase',
+  '/api/test/badSession': 'TestController.setBadSession',
+  '/api/test/loadSeasonFixture': 'TestController.loadSeasonFixture',
+  '/api/test/loadMatchFixtures': 'TestController.loadMatchFixtures',
+  '/api/test/loadFinishedGameFixtures': 'TestController.loadFinishedGameFixtures',
+  '/api/test/game': 'TestController.getGames',
+  '/api/test/match': 'TestController.getMatches',
+
+  // Catch all for Vue paths only require get for http request so we don't interfer
+  // with any blueprints routes
+  '/*': {
+    controller: 'ViewController',
+    action: 'serveIndex',
+    skipAssets: true,
+    skipRegex: /^\/api\/.*$/ // ignore /api routes
+  },
```

Comment:
We skip the `/api/*` namespace here so it isn't part of the catch all logic that goes to the Vue `index.html`.